wordpress-mobile/WordPress-iOS
# Likes: Correctly size user profile sheet pop over on iPad.

Closes #16244 

This PR applies changes to correctly size the likes user profile sheet when shown in an popover on the iPad. 

To test:
View the patch on an iPad (real or simulator). 
View different user profiles and confirm the popover is correctly sized to match.
View other bottom sheet popovers shown in the reader and when publishing and confirm there is no regression in sizing those popovers.

## Regression Notes
1. Potential unintended areas of impact
Bottom sheets presented by the Reader menu (e.g. following and p2 list) and when publishing posts.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested bottom sheets in the reader and when publishing in the simulators (iPhone and iPad) and confirmed they were sized correctly. 

3. What automated tests I added (or what prevented me from doing so)
None. 

PR submission checklist:
